media-libs/mesa-amber/mesa-amber-21.3.9-r1.ebuild

```diff
@@ -18,7 +18,7 @@ if [[ ${PV} == 9999 ]]; then
 	inherit git-r3
 else
 	SRC_URI="https://archive.mesa3d.org/${MY_P}.tar.xz"
-	KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~x64-solaris"
+	KEYWORDS="~alpha amd64 arm arm64 ~ia64 ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~x64-solaris"
```

Comment:
The commit message seems like you wanted to destable (-> ~hppa) and not dekeyword (-> **)

I don't really care, just making sure you did what you wanted :)

Reply:
Ah, oops. I'll fix that. Thanks for catching the mistake!